clients/go/admin/config.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	Endpoint              config.URL      `json:"endpoint" pflag:",For admin types, specify where the uri of the service is located."`
 	UseInsecureConnection bool            `json:"insecure" pflag:",Use insecure connection."`
 	InsecureSkipVerify    bool            `json:"insecureSkipVerify" pflag:",InsecureSkipVerify controls whether a client verifies the server's certificate chain and host name. Caution : shouldn't be use for production usecases'"`
+	CACertFilePath        string          `json:"caCert" pflag:",Use specified certificate file to verify the admin server peer."`
```

Comment:
nit: should the json also match the variable name? e.g. `caCertFilePath` ?

Reply:
Done. Had kept caCert to be in sync with flyte-cli

Reply:
ah I see, sorry to be annoying!